processor/src/operations/mod.rs

```diff
@@ -173,68 +173,72 @@ where
 }
 
 #[cfg(test)]
-impl Process<super::DefaultHost<super::MemAdviceProvider>> {
-    // TEST METHODS
-    // --------------------------------------------------------------------------------------------
-
-    /// Instantiates a new blank process for testing purposes. The stack in the process is
-    /// initialized with the provided values.
-    fn new_dummy(stack_inputs: super::StackInputs) -> Self {
-        let host = super::DefaultHost::default();
-        let mut process =
-            Self::new(Kernel::default(), stack_inputs, host, super::ExecutionOptions::default());
-        process.execute_op(Operation::Noop).unwrap();
-        process
-    }
+pub mod tests {
```

Comment:
The issue that arose was that the `#[cfg(test)]` code had imports that were not used in non-test context, and so I was getting "unused imports" warnings. 

Either we mark each with `#[cfg(test)]`, or simply create a submodule and keep all imports separate. I personally prefer keeping them cleanly separated, especially since it removes any future "unused import" warnings with future refactorings.

Reply:
Ah - makes sense. Then one small nit: let's change the name of the module from `tests` to `testing` (we use `testing` in other crates to contain test-specific utilities).

Reply:
fixed